torch/testing/_internal/common_methods_invocations.py

```diff
@@ -865,6 +907,24 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            supports_tensor_out=False,
            test_inplace_grad=False,
            sample_inputs_func=sample_inputs_broadcast_to),
+    TriangularOpInfo('cholesky_inverse',
+                     op=torch.cholesky_inverse,
+                     dtypes=floating_and_complex_types(),
+                     test_inplace_grad=False,
+                     supports_tensor_out=True,
+                     decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+                     skips=(
+                         # RuntimeError: cholesky_inverse does not support automatic differentiation for outputs with complex dtype.
+                         SkipInfo('TestGradients', 'test_fn_grad',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestGradients', 'test_fn_gradgrad',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestCommon', 'test_variant_consistency_jit',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestCommon', 'test_variant_consistency_eager',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         # These tests do not take into account custom op.get_op()
```

Comment:
Would you file an issue for this and then add a link to that issue to this comment? We should work out this pattern.

Reply:
I looked at this test more carefully now, the comment was misleading. When the scripted jit function is created inside the test it ignores `op.get_op()` because all it cares about is the name of `op` to build the script. I added `test_scripted_jit=True/False` to OpInfo to be able to skip comparing to scripted function. Is it okay?

Reply:
I was wrong about this being difficult to fix. It can be fixed by adding `op.input_func` and using it in `runAndSaveRNG` in `common_jit.py`.
I filed an issue for this https://github.com/pytorch/pytorch/issues/50837